LondonBritishLibrary/orient/BLorient581.xml

```diff
@@ -5,7 +5,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title>Life of Gabra Manfas Qǝdus, Gubāʾe Malkǝʾ, ʾƎgzer Nags</title>
+                <title>Life of Gabra Manfas Qǝddus, Gubāʾe Malkǝʾ, ʾƎgzer Nags</title>
```

Comment:
nags or nagś?

Reply:
You're right, then I think I'll actually standardise everywhere to ʾƎgziʾabəḥer nagśa. I'll do it in a separate step once all these PRs are done to avoid chaos.